sshsig.c

```diff
@@ -555,6 +558,55 @@ hash_file(int fd, const char *hashalg, struct sshbuf **bp)
 	return r;
 }
 
+#ifdef WINDOWS
+int
+sshsig_sign_fd(struct sshkey *key, const char *hashalg,
```

Comment:
same comment as earlier, can you please have the same function (instead of duplicating) and have the windows specific logic under #ifdef WINDOWS block.

Reply:
Implemented in 3dd1a019. Please note that this probably reduces the likelihood of these changes being merged upstream, as the preference there is for fewer ifdefs where possible. I don't speak for the OpenSSH project, though -- this is pure speculation on my part.